src/core/selectors.ts

```diff
@@ -62,10 +87,10 @@ namespace Selectors {
               : { navigationName: nav.field, type: 'Value', value }), [])), []);
 
   export const navigation = (state: Store.State, navigationId: string) =>
-    state.data.navigations.byId[navigationId];
+    state.data.present.navigations.byId[navigationId];
 
   export const navigations = (state: Store.State) =>
-    state.data.navigations.allIds.map((id) => Selectors.navigation(state, id));
+    state.data.present.navigations.allIds.map<Store.Navigation>(Selectors.navigation.bind(null, state));
```

Comment:
I wanted to be able to call a version of `Selectors.navigation()` that already had `state` provided. Just binding `null` as `this` because we need to pass something for `this`